connect-wallet/pages/MetaplexProvider.js

```diff
@@ -0,0 +1,21 @@
+import { Metaplex, walletAdapterIdentity, walletOrGuestIdentity, bundlrStorage } from '@metaplex-foundation/js';
+import { MetaplexContext } from './useMetaplex';
+import { useConnection, useWallet } from '@solana/wallet-adapter-react';
+import { useMemo } from 'react';
+
+export const MetaplexProvider = ({ children }) => {
+  const { connection } = useConnection();
+  const wallet = useWallet();
+
+  const metaplex = useMemo(() => {
+    return Metaplex.make(connection)
+      .use(walletOrGuestIdentity(wallet.connected ? wallet : null))
+      .use(bundlrStorage());
+  }, [connection, wallet]);
```

Comment:
That looks great!

Let's remove the `bundlrStorage` plugin since it's already the default plugin used and it will help the reader focus on the wallet.

I've also remove the import of the previous unused `walletAdapterIdentity` plugin.

```suggestion
import { Metaplex, walletOrGuestIdentity } from '@metaplex-foundation/js';
import { MetaplexContext } from './useMetaplex';
import { useConnection, useWallet } from '@solana/wallet-adapter-react';
import { useMemo } from 'react';

export const MetaplexProvider = ({ children }) => {
  const { connection } = useConnection();
  const wallet = useWallet();

  const metaplex = useMemo(() => {
    return Metaplex.make(connection)
      .use(walletOrGuestIdentity(wallet.connected ? wallet : null));
  }, [connection, wallet]);
```